social_network/accounts.py

```diff
@@ -1,13 +1,27 @@
 
 class User(object):
     def __init__(self, first_name, last_name, email):
-        pass
+        self.first_name = first_name
+        self.last_name = last_name
+        self.email = email
+        
+        self.following = []
+        self.posts = []
 
     def add_post(self, post):
-        pass
+        self.posts.append(post)
+        post.set_user(self)
 
     def get_timeline(self):
-        pass
+        posts = []
+        for user in self.following:
+            posts += user.posts
+        
```

Comment:
Nice solution for looping over posts! I do not see any code for sorting the timeline based on the posts' timestamps. (Granted, there were no tests in the file for that, but ReadMe did mention it.)